apps/src/templates/sectionProgressV2/progressV2FeedbackRedux.js

```diff
@@ -0,0 +1,106 @@
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+
+const PROGRESS_V2_FEEDBACK_URL = '/new_feature_feedback';
+const progressV2FeedbackKey = 'progress_v2';
+const PROGRESS_V2_FEEDBACK_FETCH_REQUEST = 'progressv2/feedback/fetchRequest';
+const PROGRESS_V2_FEEDBACK_FETCH_SUCCESS = 'progressv2/feedback/fetchSuccess';
+const PROGRESS_V2_FEEDBACK_FETCH_FAILURE = 'progressv2/feedback/fetchFailure';
+const PROGRESS_V2_FEEDBACK_CREATE_REQUEST = 'progressv2/feedback/createRequest';
+const PROGRESS_V2_FEEDBACK_CREATE_SUCCESS = 'progressv2/feedback/createSuccess';
+const PROGRESS_V2_FEEDBACK_CREATE_FAILURE = 'progressv2/feedback/createFailure';
+
+const initialState = {
+  isLoading: false,
+  progressV2Feedback: null,
+};
+
+export default function progressV2Feedback(state = initialState, action) {
+  switch (action.type) {
+    case PROGRESS_V2_FEEDBACK_FETCH_REQUEST:
+      return {...state, isLoading: true};
+    case PROGRESS_V2_FEEDBACK_FETCH_SUCCESS:
+      return {
+        ...state,
+        isLoading: false,
+        progressV2Feedback: action.progressV2Feedback,
+      };
+    case PROGRESS_V2_FEEDBACK_FETCH_FAILURE:
+      return {...state, isLoading: false, error: action.error};
+    case PROGRESS_V2_FEEDBACK_CREATE_REQUEST:
+      return {...state, isLoading: true};
+    case PROGRESS_V2_FEEDBACK_CREATE_SUCCESS:
+      return {
+        ...state,
+        isLoading: false,
+        progressv2Feedback: action.progressV2Feedback,
+      };
+    case PROGRESS_V2_FEEDBACK_CREATE_FAILURE:
+      return {...state, isLoading: false, error: action.error};
+    default:
+      return state;
+  }
```

Comment:
This makes it clearer that the state change operation is currently the same for the fetch and create requests, perhaps making a future change to this less error-prone if the change is inadvertently applied to one place and not the other.

```suggestion
  switch (action.type) {
    case PROGRESS_V2_FEEDBACK_FETCH_REQUEST:
    case PROGRESS_V2_FEEDBACK_CREATE_REQUEST:
      return {...state, isLoading: true};
    case PROGRESS_V2_FEEDBACK_FETCH_SUCCESS:
    case PROGRESS_V2_FEEDBACK_CREATE_SUCCESS:
      return {
        ...state,
        isLoading: false,
        progressV2Feedback: action.progressV2Feedback,
      };
    case PROGRESS_V2_FEEDBACK_FETCH_FAILURE:
    case PROGRESS_V2_FEEDBACK_CREATE_FAILURE:
      return {...state, isLoading: false, error: action.error};
    default:
      return state;
  }
```

Reply:
Thank you for this great suggestion. This actually found an error where I had both `progressV2Feedback` and `progressv2Feedback`

Reply:
Yay! I didn't even notice that initially, but a bias toward DRY definitely saves the day sometimes.